editor.planx.uk/src/@planx/components/MapAndLabel/Public/Context.tsx

```diff
@@ -198,7 +206,10 @@ export const MapAndLabelProvider: React.FC<MapAndLabelProviderProps> = (
 
   const copyFeature = (sourceIndex: number, destinationIndex: number) => {
     const sourceFeature = formik.values.schemaData[sourceIndex];
-    formik.setFieldValue(`schemaData[${destinationIndex}]`, sourceFeature);
+    formik.setFieldValue(`schemaData[${destinationIndex}]`, {
+      ...sourceFeature,
+      label: `${destinationIndex + 1}`,
```

Comment:
The label was being assigned based on the object you were copying from rather than the object we were copying to

Reply:
Good catch but I don't think this is quite working in all scenarios correctly yet! 

For example: 
- I add two trees on the map
- I fill out tree two tab
- On tab 1, I "copy from" tree 2
- I "continue" & my labels are correct
- I come "back" and "remove tree" from tab 1 - this shifts the map & tab labels for previous "tree 2" to 1 now
- I "continue" again and my passport data has a single feature with `label: 2` when it should be `label: 1` to reflect map/tab changes after a point was removed !

Reply:
Thanks so much for catching this, I'll add some adjustments